firefox-ios/Client/Frontend/Summary/SummarizerNimbusUtils.swift

```diff
@@ -21,7 +38,7 @@ struct SummarizerNimbusUtils: FeatureFlaggable {
         #endif
     }
 
-    func isHostedSummarizerEnabled() -> Bool {
+    private func isHostedSummarizerEnabled() -> Bool {
```

Comment:
I didn't see this being actively used, so I moved it back to private. cc: @FilippoZazzeroni @issammani